protoc-gen-connecpy/scripts/generate_wheels.py

```diff
@@ -14,51 +14,58 @@ def main() -> None:
             continue
         # Check https://go.dev/wiki/MinimumRequirements#operating-systems for
         # minimum OS versions, especially MacOS
-        platform = ""
+        platforms = []
         match artifact["goos"]:
             case "darwin":
                 match artifact["goarch"]:
                     case "amd64":
-                        platform = "macosx_11_0_x86_64"
+                        platforms = ["macosx_11_0_x86_64"]
                     case "arm64":
-                        platform = "macosx_11_0_arm64"
+                        platforms = ["macosx_11_0_arm64"]
             case "linux":
                 match artifact["goarch"]:
                     case "amd64":
-                        platform = "manylinux2_17_x86_64.manylinux_2014_x86_64.musllinux_1_1_x86_64"
```

Comment:
Sorry about that - I don't think the issue is multiple tags, just there is a missing underscore in `manylinux_2_17`. Can we try again with the multi-tag/single-file and fixing that?

Reply:
BTW I'm following uv's filename, but just had that typo

https://pypi.org/project/uv/#files